docs/ROADMAP.md

```diff
@@ -0,0 +1,86 @@
+# Product Roadmap
+
+MAJOR CAVEAT: This is a WORK IN PROGRESS with many items TBD. We will want weigh-in from key folks across all areas, but the idea is that this is a living, breathing document that will grow over time.
+
+### Summary
+
+There is an urgent, global need for an official WHO App to help contain and mitigate COVID-19. The app should support the WHO efforts and be available on a worldwide basis, with information tailored to the user’s location, culture and language. The app should also be sensitive to national needs and respect privacy. Once established, the system can be leveraged to proactively detect, respond early and prevent future pandemics.
+
+## Milestones
+
+Version numbering terminology:
+
+* **v0.1**: refers to the initial buildout that will be submitted to the app stores, aiming for Tuesday, March 24, 2020 ([v0.1 milestone](https://github.com/WorldHealthOrganization/app/milestone/2))
+* **v0.2+**: any versions starting with major version v0 are not intended for public distribution or consumption
+* **v1.0**: the first version that we feel offers enough value to be published onto the app stores for public download will be v1.0
+
+### v0.1
+
+**Goal:** To get a functional build of an application that can be submitted to the Apple App Store and Google Play Store for approval. This app will _NOT_ be published into those app stores for public download, but will instead serve as an approved base for future versions and to prove out that we can deliver on the technical side. The app will serve static content manually adapted from the WHO's site in English.
+
+**Target Date:** Tuesday, March 24, 2020
+
+**GitHub Milestone:** https://github.com/WorldHealthOrganization/app/milestone/2
+
+Technical Features / Requirements:
+
+* App built to be submitted on Apple App Store and Google Play Store, including code signing process
+* UI styled to exact [design spec in Figma](https://www.figma.com/file/fjzTIloCEK4FpbyDiTLj2X/iOS-UX) ([#186](https://github.com/WorldHealthOrganization/app/issues/186))
```

Comment:
@hspinks - GitHub makes your life simple than writing a long url like that. All you need is the #186 reference and it will automatically create the link for you to the issue. You can test this by looking at the "Preview" tab.

https://help.github.com/en/github/writing-on-github/autolinked-references-and-urls#issues-and-pull-requests

Repeat this many times below

```suggestion
* UI styled to exact [design spec in Figma](https://www.figma.com/file/fjzTIloCEK4FpbyDiTLj2X/iOS-UX) #186
```

Reply:
sadly, I don't think GitHub autolinks to issues/PRs in markdown files checked into the repo, only in the text of issues/PRs themselves (see the note in [the GitHub help doc](https://help.github.com/en/github/writing-on-github/autolinked-references-and-urls#issues-and-pull-requests) you linked to)... too bad cause that would be a huge time saver. Fortunately, GitHub automatically formats the markdown file for you when you view it in the browser!